internal/cmd/controller/operator.go

```diff
@@ -121,21 +121,6 @@ func start(
 
 	sched := quartz.NewStdScheduler()
 
-	if !disableGitops {
```

Comment:
We can also remove the variable, command line argument and switch in the deployment.yaml.
All git ops functionality is now in the gitjob deployment?

Reply:
Stuff related with disabling gitops in `fleet-controller` deleted.